main.py

```diff
@@ -0,0 +1,11 @@
+import random 
+
+if __name__ == "__main__":
+   while True:     
+      print(''' 1. roll the dice             2. exit     ''')    
+      user = int(input("what you want to do\n"))     
+      if user==1:         
+         number = random.randint(1,6)         
```

Comment:
It looks like the program rolls a 6-sided dice, but it isn't specified in the comment or documented. Is the plan for it always to be a 6 sided dice?